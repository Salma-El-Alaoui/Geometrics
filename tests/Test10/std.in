C c1 1 1 1
R r1 1 1 1 1
OA oa1 c1 r1
DELETE oa1
LIST
OA oa2 c1 r1
DELETE c1
LIST
DELETE r1
LIST
EXIT
