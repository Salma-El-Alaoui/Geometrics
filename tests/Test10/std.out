OK
#Element ajoute: c1
OK
#Element ajoute: r1
OK
#Element ajoute: oa1
#Element oa1 supprime
OK
C c1 1 1 1
R r1 1 1 1 1
OK
#Element ajoute: oa2
#Element c1 supprime
OK
OA oa2 r1
R r1 1 1 1 1
#Element r1 supprime
OK
OA oa2
